designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala

```diff
@@ -269,6 +277,14 @@ class ProcessesResources(
     deploymentService.getInternalProcessState(process).map(state => process.copy(state = Some(state)))
   }
 
+  private def fetchAllStateDefinitions(implicit user: LoggedUser): List[UIStateDefinition] = {
```

Comment:
As we've spoken -> move this logic to service.

Reply:
Moved that logic to service.